src/store/actions/action.types.ts

```diff
@@ -104,6 +104,20 @@ export interface DeleteMember {
   member: APIPerson;
 }
 
+export interface Authentication {
```

Comment:
If you use name SetRoles, you have to use for example setUthenticationState or uthenticationHandler etc.